src/components/footer/ChatICO.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from "react";
+import styled from "@emotion/styled";
+import { RouteComponentProps } from "@reach/router";
+
+const LatoBold = styled.div`
+  font-size: 10px;
+  font-family: Lato;
+  font-weight: bold;
+  font-style: normal;
+  font-color: black;
+`;
+
+const ChatInfo = styled.div`
+  width: 242px;
+  height: 36px;
+  display: inline-block;
+  padding-left: 24px;
+`;
+
+const Lato = styled.div`
```

Comment:
repeating component code
what about creating a component once and importing it?